state/futures/actions.ts

```diff
@@ -1321,3 +1321,22 @@ const monitorAndAwaitTransaction = async (
 	await tx.wait();
 	dispatch(updateTransactionStatus(TransactionStatus.Confirmed));
 };
+
+export const getClosePositionOrderFee = createAsyncThunk<Wei, void, ThunkConfig>(
+	'futures/getClosePositionOrderFee',
+	async (_, { getState, extra: { sdk } }) => {
+		const state = getState();
+		const position = selectPosition(state);
+		const marketInfo = selectMarketInfo(state);
+		try {
+			if (marketInfo) {
```

Comment:
Should we just check if a position exists here?

Reply:
Good point. Pushed a change that checks if a position exists before computing.